src/event-consultation-app/app.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Can't we have the app in `src/app` directly? Having it in `src/event-consultation-app` is useless no?